cleanDockerImages/cleanDockerImages.properties

```diff
@@ -0,0 +1 @@
+dockerRepos = [ "maven-remote-repo"]
```

Comment:
@madhuredde this properties file could be used to store the repositories configuration instead of an array inside the plugin code.